buildspec.yml

```diff
@@ -0,0 +1,64 @@
+version: 0.2
+
+#builds access.analyser project in aws code build
+
+#env:
+  #variables:
+     # key: "value"
+     # key: "value"
+  #parameter-store:
+     # key: "value"
+     # key: "value"
+  #secrets-manager:
+     # key: secret-id:json-key:version-stage:version-id
+     # key: secret-id:json-key:version-stage:version-id
+  #exported-variables:
+     # - variable
+     # - variable
+  #git-credential-helper: yes
+
+phases:
+  install:
+    #If you use the Ubuntu standard image 2.0 or later, you must specify runtime-versions.
+    #If you specify runtime-versions and use an image other than Ubuntu standard image 2.0, the build fails.
+    runtime-versions:
+       dotnet: latest
```

Comment:
Kontrolowałeś czy `latest` w tym przypadku na pewno oznacza dotneta 3.1? Bo już kilka razy spotkałem się z tym, że `latest` == 2

Reply:
Tak, jeśli maszynę do budowania weźmie się Amazon Linux 2 standard:3.0
https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html